PrebidMobile/Host.swift

```diff
@@ -19,9 +19,11 @@ import UIKit
 @objc public enum PrebidHost: Int {
     
     /// URL [https://ib.adnxs.com/openrtb2/prebid](URL)
+    @available(*, deprecated, message: "This property is deprecated. In the upcoming major release, the property will be removed.")
```

Comment:
Maybe we should also deprecate [Prebid.prebidServerHost](https://github.com/prebid/prebid-mobile-ios/blob/9947afd4de926df4608329d8ac208daba7e5f073/PrebidMobile/ConfigurationAndTargeting/Prebid.swift#L92) ? I don't think we'll need it anymore.